packages/app-runtime/test/lib/MockUIBridge.matchers.ts

```diff
@@ -11,12 +12,12 @@ expect.extend({
             return { pass: false, message: () => "The method showDeviceOnboarding was not called." };
         }
 
-        const matchingCalls = calls.filter((x) => x.deviceOnboardingInfo.id === deviceId);
+        const matchingCalls = calls.filter((x) => predicate(x.deviceOnboardingInfo));
         if (matchingCalls.length === 0) {
             return {
                 pass: false,
                 message: () =>
-                    `The method showDeviceOnboarding was called, but not with the specified device id '${deviceId}', instead with ids '${calls.map((e) => e.deviceOnboardingInfo.id).join(", ")}'.`
+                    `The method showDeviceOnboarding was called, but not with the specified predicate, instead with ids '${calls.map((e) => e.deviceOnboardingInfo.id).join(", ")}'.`
```

Comment:
I think it's not enough to display the IDs here, as we can filter for everything in the predicate.